### PR TITLE
fix: Support true string variants

### PR DESCRIPTION
**Pull request by erezrokah**
Fixes https://github.com/cloudquery-policies/gcp/issues/19

### CODE REVIEW

File: queries/compute/instances_without_block_project_wide_ssh_keys.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
 SELECT project_id, name, self_link AS link
 FROM gcp_compute_instances
 WHERE metadata_items IS NULL OR metadata_items ->> 'block-project-ssh-keys' IS NULL
-    OR metadata_items ->> 'block-project-ssh-keys' != 'true';
+    OR NOT metadata_items ->> 'block-project-ssh-keys' = ANY ('{true,True,TRUE}');
```
Comment (disq) [inline queries/compute/instances_without_block_project_wide_ssh_keys.sql:4]: why not `(metadata_items ->> 'block-project-ssh-keys')::boolean`?

Comment (erezrokah) [reply queries/compute/instances_without_block_project_wide_ssh_keys.sql:4]: We could, however won't it match `t,y,yes,on` as well (https://www.postgresql.org/docs/9.1/datatype-boolean.html)?

Let me check what can return from `gcp`

Comment (erezrokah) [reply queries/compute/instances_without_block_project_wide_ssh_keys.sql:4]: https://cloud.google.com/compute/docs/troubleshooting/troubleshooting-using-serial-console#api_1
https://cloud.google.com/compute/docs/connect/restrict-ssh-keys#api_2
https://cloud.google.com/compute/docs/oslogin/set-up-oslogin#gcloud

Looks like all is uppercase. Let me try and create a few of those

Comment (erezrokah) [reply queries/compute/instances_without_block_project_wide_ssh_keys.sql]: Tested `1,true,True,TRUE,y,yes,t,on`. `t,on` are considered to be false by GCP.

**Comment by disq** [comment]
or not.

**Review by irmatov — COMMENTED** [review]
LGTM

**Pull request merged**